bin/docker-compose

```diff
@@ -56,21 +56,23 @@ function set_base_vars() {
   DOCKER_COMPOSE_FLAGS=(-f "$TOOLKIT_ROOT/lib/docker-compose.base.yml")
 
   local image_name
-  if [[ -n ${SHARELATEX_IMAGE_NAME:-} ]]; then
-    image_name="$SHARELATEX_IMAGE_NAME"
+  if [[ -n ${OVERLEAF_IMAGE_NAME:-} ]]; then
+    image_name="$OVERLEAF_IMAGE_NAME"
   elif [[ $SERVER_PRO == "true" ]]; then
     image_name="quay.io/sharelatex/sharelatex-pro"
   else
     image_name="sharelatex/sharelatex"
   fi
   export IMAGE="$image_name:$IMAGE_VERSION"
 
-  if [[ ${SHARELATEX_LISTEN_IP:-null} == "null" ]];
+  OVERLEAF_LISTEN_IP=${OVERLEAF_LISTEN_IP:-${SHARELATEX_LISTEN_IP:-}}
```

Comment:
I think we can remove this fallback (and the port one below) now that we require the rc file to be migrated.

Reply:
Done in https://github.com/overleaf/toolkit/pull/217/commits/0fee1a6f7d42baddf7396d9042017d6eb3cecb68